src/pages/category/detail/Detail.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import "./Detail.scss";
+const Detail = () => {
+  const [count, setCount] = useState(1);
+  const addCount = () => {
+    setCount(count + 1);
+  };
+  const DecCount = () => {
+    setCount(count - 1);
+  };
+  return (
```

Comment:
```suggestion
import "./Detail.scss";

const Detail = () => {
  const [count, setCount] = useState(1);
  
  const addCount = () => {
    setCount(count + 1);
  };
  
  const DecCount = () => {
    setCount(count - 1);
  };
  
  return (
```

import문과 컴포넌트의 사이, 각 함수 사이, 변수 선언부와 함수 선언부의 사이, 리턴문 직전에는 공백을 넣어주는 편이 좋습니다.
공백 역시 컨벤션이 필요한 코드입니다!